hydrolib/core/dflowfm/ext/models.py

```diff
@@ -292,7 +292,7 @@ def _get_unknown_keyword_error_manager(cls) -> Optional[UnknownKeywordErrorManag
     )
 
     _header: Literal["Meteo"] = "Meteo"
-    quantity: str = Field(alias="quantity")
+    quantity: str = Field(alias="QUANTITY")
```

Comment:
In new ext file, keywords should stay lowercase, or more correctly, lowerCamelCase
```suggestion
    quantity: str = Field(alias="quantity")
```